parm/config/gfs/config.prepoceanobs

```diff
@@ -7,15 +7,20 @@ echo "BEGIN: config.prepoceanobs"
 export OCNOBS2IODAEXEC=${HOMEgfs}/sorc/gdas.cd/build/bin/gdas_obsprovider2ioda.x
 
 export SOCA_INPUT_FIX_DIR=@SOCA_INPUT_FIX_DIR@
-export OBS_YAML_DIR=${HOMEgfs}/sorc/gdas.cd/parm/soca/obs/config
+
+export OBS_YAML_DIR=${OBS_YAML_DIR:-${PARMgfs}/gdas/soca/obs/config}
```

Comment:
I don't think there is an `OBS_YAML_DIR` defined before this point, and it shouldn't be.
```suggestion
export OBS_YAML_DIR="${PARMgfs}/gdas/soca/obs/config"
```

Reply:
This is to accommodate GDASApp ctests, which don't run the link script

Reply:
changed - it's in the CI but not the ctests, will add